docs/topics/kapt.md

```diff
@@ -70,14 +70,6 @@ Follow these steps:
    `kaptAndroidTest` and `kaptTest`. Note that `kaptAndroidTest` and `kaptTest` extends `kapt`, so you can just provide the
    `kapt` dependency and it will be available both for production sources and tests.
 
-4. To use the newest Kotlin features with kapt, for example, [repeatable annotations](annotations.md#repeatable-annotations), 
```

Comment:
@udalov, WDYT, should we still mention separately that a user can use the newest Kotlin features with kapt, for example, repeatable annotations?

Reply:
No, I don't think it's necessary